controllers/index.py

```diff
@@ -1,14 +1,20 @@
 import os
 from pathlib import Path
 
-from py_server.config.web import PROJECT_ROOT
+from config.web import PROJECT_ROOT
 
 
 class IndexController:
 
 	def get_view(self):
-		view_file = os.path.join(PROJECT_ROOT, 'py_server/view/index.html')
+		# Use the working directory, not project root
+		# Using project root assumes too many hard coded relative paths from 'dev' root 
+		view_file = os.path.join(os.getcwd(), 'view/index.html')
```

Comment:
kinda the same, I don't think where you are running the code from should be driven this, but I learned something new here